linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -420,6 +446,7 @@ object Infos {
     private val methodsCalled = mutable.Set.empty[MethodName]
     private val methodsCalledStatically = mutable.Set.empty[NamespacedMethodName]
     private val jsNativeMembersUsed = mutable.Set.empty[MethodName]
+    private val lambdaDescriptorsUsed = mutable.Set.empty[NewLambda.Descriptor]
```

Comment:
Could you add a comment explaining why this is attached to a class and not the method that actually calls `NewLambda`?